README.md

```diff
@@ -1,6 +1,26 @@
+Teleport protects access to your infrastructure.
```

Comment:
Nit: It feels a little odd to have this prose above the hero image/repository badges

Reply:
+1